iptables.yaml

```diff
@@ -70,6 +71,16 @@ pipeline:
       install -D -m644 ebtables.confd "${{targets.destdir}}"/etc/conf.d/ebtables
 
 subpackages:
+  - name: iptables-cap_set_admin
```

Comment:
What about `iptables-xtables-privileged`? Since we're only setting caps for xtables